v2/index.html

```diff
@@ -522,10 +525,51 @@ <h3>Musical Notation</h3>
         <div class="issue" data-number="24"></div>
         <div class="issue" data-number="43"></div>
         <div class="issue" data-number="56"></div>
+        <section>
+            <h4>Structure</h4>
+            <p>
+                The Musical Notation section of an encoding is given as a single line of characters. These characters
+                represent the main body of the notation encoding. To represent complex notational figures, such as
+                notes, chords, beams, or tuplets, a group of characters can act as a single <a>logical unit</a>.
+            </p>
+            <p>
+                A musical note is the single most complex logical unit within the <abbr>PAEC</abbr>. It consists of
+                one or more characters representing a note on a musical staff. Many characters representing a note
+                are optional, but where one or more character for a note occurs, they MUST occur in the following
+                order:
+            </p>
+            <figure>
+                <ol>
+                    <li>(optional) The grace note character, or the grace note group characters;</li>
+                    <li>(optional) The octave character(s);</li>
+                    <li>(optional) The duration character, followed by a duration dot character;</li>
```

Comment:
I think we need to keep the order between octave an duration flexible, but maybe this is for a later fix?

Reply:
Do you have a use case where keeping the order flexible is useful?

Reply:
No. It is just than when typing it requires additional thinking to have a fix order. I would understand people would complain about it since. Also, there is no strong argument for having the octave before the duration - one might argue that octave relates to the pitch, so that duration should come first.

Reply:
I agree that we need to simplify data processing as much as possible. That would be the only exception for me from my (little) experience in cataloguing incipit. Maybe @jenniferward has a opinion on this?

Reply:
I have been paying attention to the order of octave vs. duration for a while now and I do find that sometimes my brain says "OK, first octave" and sometimes it says "OK, first duration".  I think it happens when there is a significant switch: a big octave change or a big rhythmic change or if I have to coordinate one of them with other elements (curly brackets etc.). But I haven't kept track of when this happens, it's just my feeling.

Can we take a look at the data and find out how often `'''4` vs. `4'''` (or similar) happens? I think we can get used to a consistent order if prompted but I'd be curious what the natural tendency is.

Reply:
I think this is the main point. It really depends on the content of the music. Sometimes it clearly makes more sense to have the octave first, and sometimes the other way around. Running an analysis to see if there is a clear tendency towards octave-first would be good.

Reply:
I meant that my fingers go naturally to either the octave first or the duration first, depending on musical context.